src/wrappers/themis/jsthemis/secure_cell_context_imprint.cpp

```diff
@@ -55,18 +71,46 @@ namespace jsthemis {
   }
 
   void SecureCellContextImprint::encrypt(const Nan::FunctionCallbackInfo<v8::Value>& args) {
+    themis_status_t status = THEMIS_SUCCESS;
```

Comment:
I think it's safer to init status as `THEMIS_FAIL`, because *only* successful encryption will make this status `THEMIS_SUCCESS`

```suggestion
    themis_status_t status = THEMIS_FAIL;
```